hardhat.config.ts

```diff
@@ -27,9 +27,39 @@ const {
   BSCSCAN_API_KEY,
   ARBISCAN_API_KEY,
   OPTIMISTIC_ETHERSCAN_API_KEY,
-  ZKSYNC_EXPLORER_API_KEY
+  ZKSYNC_EXPLORER_API_KEY,
+  VERSION,
+  ENV
 } = process.env;
 
+const version = VERSION ? VERSION.split('.')[0] : 'v1';
```

Comment:
throw an error if version is `undefined`

Reply:
This is causing an Error while executing tests in Github. At the time of Github test executing, we wont be giving any tag right, so the VERSION remains `undefined`.

Reply:
Addressed this.